odml/tools/rdf_converter.py

```diff
@@ -310,6 +322,32 @@ class Section.
 
         return None
 
+    def _parse_custom_subclasses(self, custom_subclasses):
+        """
+        Parses a provided dictionary of "Section type": "RDF Subclass name"
+        key value pairs and adds the pairs to the parsers' 'section_subclasses'
+        default dictionary. Existing key:value pairs will be overwritten
+        with provided custom key:value pairs and a Warning will be issued.
+        Dictionary values containing whitespaces will raise a ValueError.
+
+        :param custom_subclasses: dictionary of "Section type": "RDF Subclass name" key value pairs.
+                                  Values must not contain whitespaces, a ValueError will be raised
+                                  otherwise.
+        """
+
+        # Do not allow whitespaces in values
+        if " " in "".join(custom_subclasses.values()):
```

Comment:
The `whitespace` property of the stdlib module `string` contains all ASCII whitespace characters.  Maybe we should use that to check for other kinds of whitespace instead (tab, linefeed, return, etc).